rbd-target-api.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-
```

Comment:
Drop this from the patch. It is not related. And, I do not know if it is the proper coding style, but if it is you could fix it in the other code too in a separate patch.